### PR TITLE
PlainTable support

### PR DESCRIPTION
**Pull request by critical27**
To support rocksdb PlainTable and nebula work in memory
1. Dir of rocksdb wal and nebula wal should be configurable
2. Backup rocksdb periodically, and try to recover from latest backup
3. Fix the 2.0 prefix bloom filter

### CODE REVIEW

File: src/kvstore/RocksEngine.cpp [hunk 1]
```diff
@@ -114,6 +125,8 @@ RocksEngine::RocksEngine(GraphSpaceID spaceId,
     db_.reset(db);
     partsNum_ = allParts().size();
     LOG(INFO) << "open rocksdb on " << path;
+
+    backup();
```
Comment (bright-starry-sky) [inline src/kvstore/RocksEngine.cpp:129]: A little confused, seems like a run for nothing, could give some comments?

Comment (critical27) [reply src/kvstore/RocksEngine.cpp:129]: Long story, I can't remember very clear.
1. If we don't backup here, we write some data, and storage is stopped before we do a valid backup.
2. Mock machine reboot, so rocksdb data in memory is lost. (rocksdb wal is persisted in disk, so we could recover from it in theory)
3. Restart the storage, rocks db will be failed open, log is `While creating a new Db, wal_dir contains existing log file: ...`.

I believe the problem is that if we backup here, it is not nothing here. Something like `CURRENT` and `MANIFEST` will be backuped. In that way, rocksdb will think the wal is legal.

Comment (bright-starry-sky) [reply src/kvstore/RocksEngine.cpp:129]: I see. It's a very complicated story.

**Review by bright-starry-sky — DISMISSED** [review]
Good job. Well done.

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by panda-sheep — APPROVED** [review]
Woohoo，Great job！
I learned a lot from it.👍

**Pull request merged**